### PR TITLE
Add unit test for cloudfront command

### PR DESCRIPTION
**Pull request by jamesls**
This will prevent future regressions against basic
cloudfront commands.

cc @kyleknap @danielgtaylor

### CODE REVIEW

File: tests/unit/cloudfront/test_get_distribution_config.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# language governing permissions and limitations under the License.
+from awscli.testutils import BaseAWSPreviewCommandParamsTest as \
+    BaseAWSCommandParamsTest
+
+
+class TestListDistributions(BaseAWSCommandParamsTest):
+
+    prefix = 'cloudfront get-distribution-config'
+
+    def test_list_distributions(self):
```
Comment (kyleknap) [inline tests/unit/cloudfront/test_get_distribution_config.py]: whoops. I messed up the naming for the class and the test method. Should be ``TestGetDistributionConfig`` and ``test_get_distribtuion_config``

**Comment by kyleknap** [comment]
Had a small comment after reviewing the test I wrote. Otherwise, :ship:

File: tests/unit/cloudfront/test_get_distribution_config.py (hunk 1 above)
Comment (jamesls) [inline tests/unit/cloudfront/test_get_distribution_config.py]: Also, what about the BaseAWSPreviewCOmmandParamsTest?

The test actually passes for me as is just using the BaseAWSCommandParamsTest.

Comment (jamesls) [inline tests/unit/cloudfront/test_get_distribution_config.py]: Nevermind completely misread the import statement.

Comment (kyleknap) [inline tests/unit/cloudfront/test_get_distribution_config.py]: Yeah if you do not configure cloudfront to preview in the config and use BaseAWSCommandParamsTest, it fails

**Pull request merged**